README.adoc

```diff
@@ -1,15 +1,21 @@
-= Kiali image:https://travis-ci.org/kiali/kiali.svg["Build Status", link="https://travis-ci.org/kiali/kiali"]
+= Kiali image:https://img.shields.io/twitter/url/http/shields.io.svg?style=social["Tweet about Kiali", link="https://twitter.com/intent/tweet?text=Learn%20what%20your%20Istio-Mesh%20is%20doing.%20Visit%20https://www.kiali.io/%20and%20@kiali_project"]
 :toc: macro
```

Comment:
line ~31 below mentions kiali.org rather than kiali.io, I guess it should be aligned to either one of the options in all places :)